public_html/wp-content/plugins/pattern-directory/includes/notifications.php

```diff
@@ -0,0 +1,234 @@
+
+	$pattern_title = get_the_title( $post );
+
+	if ( $locale ) {
+		switch_to_locale( $locale );
+	}
+
+	$reasons = array();
+	$flags = get_posts( array(
+		'post_type' => FLAG,
+		'post_parent' => $post->ID,
+		'post_status' => PENDING_STATUS,
+	) );
+	if ( ! empty( $flags ) ) {
+		foreach ( $flags as $flag ) {
+			$terms = get_the_terms( $flag, REASON );
+			if ( is_array( $terms ) ) {
+				$reasons += $terms;
+			}
+		}
+		$reasons = array_map(
+			function( \WP_Term $reason ) {
+				return $reason->name;
+			},
+			$reasons
+		);
+		$reasons = array_unique( $reasons );
+		// translators: used between list items, there is a space after the comma.
+		$reasons = implode( __( ', ', 'wporg-patterns' ), $reasons );
+	} else {
+		// If it doesn't have flags, it must have gotten here by getting marked as spam.
+		$reasons = __( 'Spam', 'wporg-patterns' );
+	}
+
+	$subject = esc_html__( 'Pattern being reviewed', 'wporg-patterns' );
+
+	$message = sprintf(
+		// translators: Plaintext email message. Note the line breaks. 1. Pattern title; 2. Pattern URL;
+		esc_html__( 'Hi there!
+
+Thanks for submitting your pattern. Unfortunately, your pattern, %1$s, has been flagged for review due to the following reason(s):
+
+%2$s
+
+Your pattern has been unpublished from the Block Pattern Directory at this time, and will receive further review. If the pattern meets the guidelines, we will re-publish it to the Block Pattern Directory. Thanks for your patience with us volunteer reviewers!', 'wporg-patterns' ),
+		esc_html( html_entity_decode( $pattern_title ) ),
+		esc_html( html_entity_decode( $reasons ) )
+	);
+
+	if ( $locale ) {
+		restore_current_locale();
+	}
+
+	send_email( $email, $subject, $message );
+}
+
+/**
+ * Notify when a pattern has been unlisted.
+ *
+ * @param \WP_Post $post
+ *
+ * @return void
+ */
+function notify_pattern_unlisted( $post ) {
+	$author = get_user_by( 'id', $post->post_author );
+	if ( ! $author ) {
+		return;
+	}
+
+	$email  = $author->user_email;
+	$locale = get_user_locale( $author );
+
+	$pattern_title = get_the_title( $post );
+
+	$reasons = get_the_terms( $post, REASON );
+	if ( ! empty( $reasons ) ) {
+		$reason_term = reset( $reasons );
+		$reason = $reason_term->name;
+	} else {
+		$reason = __( 'Other', 'wporg-patterns' );
+	}
+
+	if ( $locale ) {
+		switch_to_locale( $locale );
+	}
+
+	$subject = esc_html__( 'Pattern unlisted', 'wporg-patterns' );
+
+	$message = sprintf(
+		// translators: Plaintext email message. Note the line breaks. 1. Pattern title; 2. Pattern URL;
+		esc_html__( 'Hello,
+
+Your pattern, %1$s, has been unlisted from the Block Pattern Directory due to the following reason:
+
+%2$s
+
+If you would like to resubmit your pattern, please make sure it follows the guidelines:
+
+%3$s', 'wporg-patterns' ),
+		esc_html( html_entity_decode( $pattern_title ) ),
+		esc_html( html_entity_decode( $reason ) ),
+		'https://wordpress.org/patterns/' // TODO: get the actual URL.
```

Comment:
The URL to the guidelines is just the About page.
```suggestion
		'https://wordpress.org/patterns/about/'
```